buildbot.mariadb.org/util/nginx.conf

```diff
@@ -1,5 +1,12 @@
 # Buildbot server configuration
-#
+# Main context
+
+worker_processes auto;
```

Comment:
These values are specified in nginx.conf instead of the sites-enabled/buildbot virtual host specific rules. I see no reason to change the defaults for now.